app/models/video.py

```diff
@@ -1,4 +1,17 @@
 from app import db
 
 class Video(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    #how many I own
+    total_inventory = db.Column(db.Integer)
+    #customers = db.relationship("Rental", back_populates="video")
```

Comment:
`Video` should have a relationship with `Rental` so this line shouldn't be commented. We want to be able to identify videos that are 'rented' :

```suggestion
        rentals = db.relationship("Rental", backref="video")
```